### PR TITLE
Add cpu option

### PR DESCRIPTION
**Pull request by MedericFourmy**
Enable running singleview cosypose (detection + pose estimation) on CPU only machine for easy debugging. The changes consist:

- Automatically decides to use the cpu in cosypose detector if no GPU present
- Add an argument to BulletBatchRenderer to run batch rendering on CPU instead of GPU

Default behavior/API is not affected by current changes when running on GPU equipped machines. 
For now, only the singleview pipeline is affected as it is expected that training, dataset-level evluation or multiview computations to be only used on GPU.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,16 @@ Winner of the <a href="https://bop.felk.cvut.cz/challenges/bop-challenge-2020/">
 </div>
 
 
+# Important Note
+<body>
+<h1 align="center">
+<p style="color:red;font-size:30px">This repository is no longer being maintained.
+<p style="color:red;font-size:30px">If you are looking to use CosyPose and encounter any issues, please see:
+<p style="color:red;font-size:30px">
+<a href="https://github.com/Simple-Robotics/cosypose">https://github.com/Simple-Robotics/cosypose</a>
```
Comment (nim65s) [inline README.md:34]: Maybe we don't need this commit in this repo :)

**Review by nim65s — CHANGES_REQUESTED** [review]
maybe we should document how this `gpu_renderer` should be set in the first place ?

File: README.md (hunk 1 above)
Comment (MedericFourmy) [reply README.md:34]: Indeed

**Comment by MedericFourmy** [comment]
`gpu_renderer` is an argument of `BaseSceneRenderer`
https://github.com/Simple-Robotics/cosypose/blob/8e284acc39b13c7a172164ad0f482e3c1c83ca0d/cosypose/rendering/bullet_scene_renderer.py#L17
which is used by `BulletBatchRenderer` to render objects using bullet. I could add some docstring in `bullet_batch_renderer.py`, but there would be many other arguments to document.

**Pull request merged**